lib/components/Input/wheel-picker/base/index.tsx

```diff
@@ -144,13 +219,26 @@ const GenericWheelPickerContent = forwardRef<
                                 )}
                             >
                                 <WheelPickerContainer
+                                    data={
+                                        data as {
+                                            value: string | number;
+                                            label?: string | ReactNode;
+                                        }[][]
+                                    }
                                     inputValues={inputValues}
                                     close={close}
                                     setSelectedValue={setSelectedValue}
+                                    is12Hour={is12Hour}
                                     setInputValues={(
                                         index: number,
                                         newValue: string | number,
                                     ) => updateItemAtIndex(index, newValue)}
+                                    selectedTime={selectedTime ?? initialValues}
```

Comment:
`selectedTime` will never be empty string, correct?

Reply:
removed